src/ocean/LibVersion.d

```diff
@@ -0,0 +1,9 @@
+module ocean.LibVersion;
+
+const has_features_2_6 = true;
```

Comment:
I still don't get it. Also, technically, this is a new feature (adding a module), so we shouldn't release that as a point release. We could make an exception, but I'd prefer not to unless there is rally a **very** strong case for it.